client/components/HomePage.tsx

```diff
@@ -1,11 +1,15 @@
 import React from "react";
 import ZoopList from './ZoopList'
+import SendZoopDialog from "./SendZoopDialog";
 
 const HomePage = () => {
-    return(
+
+   
+    return (
         <>
             <div>I am the Home Page</div>
             <ZoopList />
+            <SendZoopDialog />
```

Comment:
It's currently not on LoginPage or RegisterPage because at no point is the user on those pages logged in. Do you want it to be? It certainly would be DRYer to just add it to `App.tsx`. The button is disabled when the user is not logged in anyway.

Reply:
Button should be hidden when not logged in. I would recommend adding it in App.tsx and just not rendering it if there is no token